node-runner-cli/setup/Docker.py

```diff
@@ -46,132 +42,11 @@ def run_docker_compose_up(keystore_password, composefile, trustednode):
                           })
 
     @staticmethod
-    def setup_compose_file(composefileurl, file_location, enable_transactions=False):
```

Comment:
This method is been removed. It was dynamically weaving together yaml, but it dependented on code changes to make updates.  

Now it is moved to jinja templating which is lot easier to maintain and update for changes in docker compose or config files

Reply:
Nice change :)